session/session.go

```diff
@@ -3440,8 +3443,22 @@ func (s *session) updateTelemetryMetric(es *executor.ExecStmt) {
 	}
 }
 
+// GetBuiltinFunctionUsage returns the replica of counting of builtin function usage
 func (s *session) GetBuiltinFunctionUsage() map[string]uint32 {
-	return s.builtinFunctionUsage
+	s.functionUsageMu.RLock()
+	defer s.functionUsageMu.RUnlock()
+	replica := make(map[string]uint32)
```

Comment:
please remove map creation out of the lock scope.

Reply:
updated.